scripts/external-lookup-services.js

```diff
@@ -4863,6 +4863,33 @@ var externalLookupServices = angular.module('externalLookupServices', ['ngResour
                 });
                 return promise;
             },
+            lookupVaccine: function(fNr, kommuneNr, userId) {
+                var url = '../' + DHIS2URL + '/vaksine/sok';
+                var promise = $http({
+                    method: 'POST',
+                    url: url,
+                    data: {fnr:fNr, kommunenr:kommuneNr, userid:userId},
+                    headers: {'Content-Type': 'application/json'}
+                }).then(function(response){
+                    return response.data;
+                },function(error) {
+                    var errorMsgHdr, errorMsgBody;
+                    errorMsgHdr = $translate.instant('error');
+
+                    errorMsgBody =  'Feil ved henting av vaksinedata:' + fNr;
+
+                    if(error.status == 403) {
+                        errorMsgBody = `Tjenesten Fiks prøvesvar er ikke tilgjengelig for deg.`;
```

Comment:
Wrong service name.

Should also provide a better explanation text - something akin of the one for folkeregister search.
The end-user might get this if he/she lacks authorisation. Or alternatively if the organisation have failed to activate the service.